lib/menu.rb

```diff
@@ -0,0 +1,33 @@
+require_relative 'dish'
+
+class Menu
+
+  attr_reader :list
```

Comment:
Don't expose the internal array publicly.  Create an interface on `Menu` for the menu to be queried.